### DISCUSSION TITLE
DataGrid Feature Request: Customize Hovering Color

### DISCUSSION THREAD

**Discussion by Eve-YeShen** [vmware/clarity#5906] [Ideas]
This is a feature request for setting the row background color on hover. Essentially do add a design token for this when the Clarity Core datagrid component is on the table. 

Our use case:
In table, there are rows which are clickable and rows which are not clickable. We would like to make the hovering color different for clickable and non-clickable rows. Also, change the cursor shape for clickable rows.

**Comment by bbogdanov** [comment]
@coryrylan @colinreedmiller

**Reply by Eve-YeShen** [reply]
Considering the breaking risk and the inconvenience for the keyboard users and visual disabled users, we will no longer consider the hover and click feature on our table.

**Comment by coryrylan** [comment]
To resolve this discussion, the hover color will be exposed as a customizable part of our theming API however we won't be adding clickable rows to for accessibility.
